cmd/analyzer/analyzer.go

```diff
@@ -550,3 +586,14 @@ func Register(parentCmd *cobra.Command) {
 	analyzeCmd.Flags().StringVar(&configFile, "config", "./config/local.yml", "path to the config.yml file")
 	parentCmd.AddCommand(analyzeCmd)
 }
+
+// For testing purposes only.
+type ServiceTester struct {
```

Comment:
It's also possible to move the `TestSequencing` test into the `package analyzer`  (e.g. in sequencing_test.go file). Then the test will have access to package private fields directly and this workaround is not needed.

Reply:
Ah nice, thanks. Converted.

I had looked up a random existing test and figured that the separate `_test` package is a strong Go convention for tests. I see now that both types of testing (from `mypackage_test` and from `mypackage`) are endorsed. And sure, `mypackage_test` is a little cleaner, but ... not much, with my `ServiceTester` hack. So, not worth the hassle in this case, IMO.